internal/os/disk/types.go

```diff
@@ -109,6 +109,7 @@ type StorageIdentifier struct {
 	Identifier     [1]byte
 }
 
-type DiskPath struct {
-	Path string `json:"Path"`
+type Disk struct {
+	Path         string `json:"Path"`
+	SerialNumber string `json:"SerialNumber"`
```

Comment:
@jingxu97 @ddebroy do we also need to bump the api version?

Reply:
This is an internal type not exported through the client APIs. So no API version bump needed.

Reply:
Specifically, keeping the types of disk identifiers generic at https://github.com/kubernetes-csi/csi-proxy/blob/master/client/api/disk/v1beta2/api.proto#L71 helped.